app/models/katello/ping.rb

```diff
@@ -193,8 +193,8 @@ def pulp3_without_auth(url)
         end
 
         workers = json["online_workers"] || []
-        resource_manager_exists = workers.any? { |worker| worker["name"].include?("resource-manager@") }
-        reservered_resource_worker_exists = workers.any? { |worker| worker["name"] =~ /reserved-resource-worker-./ }
+        resource_manager_exists = workers.any? { |worker| worker["name"].include?("resource-manager") }
+        reservered_resource_worker_exists = workers.any? { |worker| worker["name"] =~ /#{Socket.gethostbyname(Socket.gethostname).first}/ }
```

Comment:
I would probably avoid trying to match the hostname (which wouldn't work if we ever support multi-box scenarios).  Maybe its good enough to just count any workers otehr than the resource-manager?

Reply:
yeah that's true, I'll just make sure there's at least one other worker